test/uk/gov/hmrc/traderservices/journey/TraderServicesFrontendFormatSpec.scala

```diff
@@ -147,6 +147,29 @@ class TraderServicesFrontendFormatSpec extends UnitSpec {
           )
         )
       )
+      validateJsonFormat(
+        """{"state":"AnswerExportQuestionsMandatoryVesselInfo","properties":{"declarationDetails":{"epu":"123","entryNumber":"Z00000Z","entryDate":"2020-10-05"},"exportQuestionsAnswers":{"requestType":"Hold","priorityGoods":"LiveAnimals","freightType":"RORO"}}}""",
+        State.AnswerExportQuestionsMandatoryVesselInfo(
+          DeclarationDetails(EPU(123), EntryNumber("Z00000Z"), LocalDate.parse("2020-10-05")),
+          ExportQuestions(
+            requestType = Some(ExportRequestType.Hold),
+            priorityGoods = Some(ExportPriorityGoods.LiveAnimals),
+            freightType = Some(ExportFreightType.RORO)
+          )
+        )
+      )
```

Comment:
Request type will need to be C1601 instead of Hold here for mandatory. 

Also, is it worth having another test for state.AnswerExportQuestions**Optional**VesselInfo?